x.toml

```diff
@@ -156,3 +156,26 @@ members = [
     "secure/key-manager",
     "storage/backup/backup-cli",
 ]
+
+# ---
+# Determinator rules
+# ---
+
+# CI-related files. TODO: maybe have separate rules for local and CI?
+[[determinator.path-rule]]
+globs = [".github/**/*", ".circleci/**/*", "codecov.yml"]
+mark-changed = "all"
+
+# Core Rust and devtools files.
+[[determinator.path-rule]]
+globs = ["rust-toolchain", "cargo-toolchain", "cargo-flags", "x.toml"]
+mark-changed = "all"
```

Comment:
This rule would be redundant as well.

Reply:
Even though it would be, per our discussion I've added a linter to ensure that every file in the repo is covered by a rule or package.